worker/docker.go

```diff
@@ -70,11 +71,11 @@ func NewDockerManager(containerImageID string, gpus []string, modelDir string) (
 	}, nil
 }
 
-func (m *DockerManager) Warm(ctx context.Context, pipeline string, modelID string) error {
+func (m *DockerManager) Warm(ctx context.Context, pipeline string, modelID string, optimizationFlags OptimizationFlags) error {
```

Comment:
@yondonfu I now decided to use the more specific `optimizationFlags` instead of the more general `flags`. We can however also change this if we think we want to bundle other container flags in the future.